k9mail/src/main/java/com/fsck/k9/helper/HtmlSanitizer.java

```diff
@@ -0,0 +1,47 @@
+package com.fsck.k9.helper;
+
+
+import org.htmlcleaner.CleanerProperties;
+import org.htmlcleaner.HtmlCleaner;
+import org.htmlcleaner.SimpleHtmlSerializer;
+import org.htmlcleaner.TagNode;
+
+
+public class HtmlSanitizer {
+    private HtmlSanitizer() {}
+
+    public static String sanitize(String html) {
+        HtmlCleaner cleaner = new HtmlCleaner();
```

Comment:
Create a single static HtmlCleaner object instead of recreating it every time.

```java
   // Clean HTML taken from simple string, file, URL, input stream,
   // input source or reader. Result is root node of created
   // tree-like structure. Single cleaner instance may be safely used
   // multiple times.
   TagNode node = cleaner.clean(...);
```

http://htmlcleaner.sourceforge.net/doc/index.html?org/htmlcleaner/TagNode.html